src/api/modules/utils.ts

```diff
@@ -134,106 +132,6 @@ export const appIdFromRegistry = (app: string, majorLocator: string) => {
     .catch(handleError(app))
 }
 
-const FREE_BILLING_OPTIONS_TYPE = 'free'
-export const isFreeApp = ({ type, free }: { type?: string; free?: boolean }) =>
-  type === FREE_BILLING_OPTIONS_TYPE || free
-
-const SPONSORED_BILLING_OPTIONS_TYPE = 'sponsored'
```

Comment:
What if the app is sponsored? I don't think we should redirect to the app store, right?

Reply:
Isn't this code related to how you solved the samsung bug @valentino-amadeus ?

Reply:
Hey, @gris! Does the comment above answer your question? 😄

Reply:
>What if the app is sponsored? I don't think we should redirect to the app store, right?

The app will be installed instantly. Notice the value change in the `termsOfUseAccepted` parameter [when `installApp` is called inside `prepareInstall`](https://github.com/vtex/toolbelt/blob/47cfbb741e0513664c665ffb9af0f2cc2a147f06/src/modules/apps/install.ts#L52).

Reply:
This is weird, @marlonguimaraes... I tried to install the same app in the same account, and it worked successfully. 🤔 

<img width="753" alt="Captura de Tela 2021-05-21 às 13 34 00" src="https://user-images.githubusercontent.com/67928158/119170296-7132f800-ba39-11eb-864a-ebc853a81d67.png">

Reply:
Have you ran `yarn run watch` before the `vtex-test`?

Reply:
Oh nice, ok, that answers my question.